org.eclipse.xtext/src/org/eclipse/xtext/validation/Issue.java

```diff
@@ -18,6 +18,25 @@
  */
 public interface Issue {
 
+	/**
+	 * @author Christian Dietrich - Initial contribution and API
+	 * @since 2.19
+	 */
+	interface IssueExtension {
+		/**
+		 * Returns the column in the end line of the issue. It's not the virtual column but literally
+		 * the character offset in the column, e.g. tab ('\t') counts as one character.
+		 * The first char in a line has column number 1, the number is one-based.
+		 * 
+		 * If no column information is available, returns -1.
```

Comment:
My gut feeling is, that `default getEndColumn() { return -1 }` would fit nicely into Issue itself.

Reply:
and what for the endlinenumber?

Reply:
a getLineNumber?